File: app/src/app/Modelos/Operacion.ts
import { EstadoOperacion } from './EstadoOperacion';
import {TipoOperacion} from './TipoOperacion';


export class Operacion {
    IdOperacion: number;
    FechaOperacion: string;
    Monto: number;
    TipoOperacion: TipoOperacion;
    EstadoOperacion: EstadoOperacion;
    CvuHasta: string;
  }

File: ProyectoClipMoney2020/Models/Clases/Operacion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProyectoClipMoney2020.Models
{
    public class Operacion
    {
        public long idOperacion { get; set; }
        public DateTime fechaOperacion { get; set; }
        public decimal monto { get; set; }
        public TipoOperacion tipoOperacion { get; set; }
        public EstadoOperacion estadoOperacion { get; set; }
        public string cvuHasta { get; set; }
        public string cvuDesde { get; set; }
    }
}